readme.md

```diff
@@ -17,9 +17,16 @@ Mycroft Voice Satellite, connect to  [Mycroft HiveMind](https://github.com/Jarba
 
 ![](./voice_sat.png)
 
-
 ## Install
 
+Install dependencies (if needed)
```

Comment:
Add a note for inexperienced users that this is for Debian/Ubuntu/Raspberry Pi, and 👍 thanks

We'll presumably get in there and add the same for Arches and Carmen Sandiegos.